src/python/WMQuality/Emulators/DBSClient/MockDbsApi.py

```diff
@@ -35,6 +38,16 @@
 mockData['https://cmsweb-prod.cern.ch/dbs/prod/global/DBSReader'] = mockDataGlobal
 mockData['https://cmsweb-prod.cern.ch/dbs/prod/phys03/DBSReader'] = mockData03
 
+def _unicode(data):
```

Comment:
Dario, how about we create a GH issue asking for this function (and its references) to be removed once we have services running in python3 only?

Reply:
Yeah, this is a nice suggestion! I can also create a new tag for such issues, like `clean py3`, `followup py3` or whatever else you like

Reply:
created #10322 for addressing this